component/namespace-policies.jsonnet

```diff
@@ -58,6 +58,36 @@ local organizationNamespaces = kyverno.ClusterPolicy('organization-namespaces')
     validationFailureAction: 'enforce',
     background: false,
     rules: [
+      {
+        name: 'set-default-organization',
+        match: {
+          resources: {
+            kinds: [
+              'Namespace',
+            ],
+          },
+        },
+        context: [
+          {
+            name: 'ocpuser',
+            apiCall: {
+              urlPath: '/apis/user.openshift.io/v1/users',
+              jmesPath: "items[?metadata.name=='{{ request.userInfo.username}}']",
+            },
+          },
+        ],
+        exclude: common.BypassNamespaceRestrictionsSubjects(),
+        mutate: {
+          patchStrategicMerge: {
+            metadata: {
+              labels: {
+                '+(appuio.io/organization)':
+                  '{{ocpuser[0].metadata.annotations."appuio.io/default-organization"}}',
+              },
+            },
+          },
+        },
+      },
       {
         name: 'has-organization',
         match: {
```

Comment:
I think this rule is redundant now. So we could just remove it

Reply:
The rule is still useful to catch cases where a user erroneously doesn't have a default organization assigned, since the mutate rule doesn't trigger a failure if it doesn't find the value, but instead just doesn't add the label.